### ISSUE TITLE
Unable to mark release as Dead

### ISSUE THREAD

**Issue by Stoyvo** [atlassian/nucleus#75]
There was a release identified as v0.0.0-alpha.19b, and now release v0.0.0-alpha.20 is available. Cannot mark .19b as dead.

**Comment by MarshallOfSound** [comment]
@Stoyvo Is `v0.0.0-alpha.20` at 100% rollout?

**Comment by Stoyvo** [comment]
@MarshallOfSound Yes. I manually updated the sqlite3 db to make the change, but for some reason the UI thinks its the most recent so Mark as Dead isn't an option is my best guess. It's ID in the database is less than the most recent
